tests/test_api/test_subscriptions.py

```diff
@@ -297,3 +297,66 @@ def test_update_subscription_date_too_far_out(client, tables):
                                 json={'end': end})
     assert api_response.status_code == HTTPStatus.BAD_REQUEST
     assert f'End date: {end.isoformat(timespec="seconds")} must be within 180 days' in api_response.json['detail']
+
+
+def test_get_subscription_by_id(client, tables):
```

Comment:
should include a test for the 404 not found case here, as well as for `test_update_subscription`